ecs.php

```diff
@@ -13,7 +15,11 @@
 
     $parameters->set(
         Option::SKIP,
-        ['SlevomatCodingStandard\Sniffs\Exceptions\ReferenceThrowableOnlySniff.ReferencedGeneralException' => ['tests/Exception/*.php']]
+        [
+            'SlevomatCodingStandard\Sniffs\Exceptions\ReferenceThrowableOnlySniff.ReferencedGeneralException' => ['tests/Exception/*.php'],
+            OperatorSpacingSniff::class => null,
```

Comment:
I guess this is about spaces around union types; is there a related issue in php codesfniffer?

Reply:
it doesn't support php 8 yet